### ISSUE TITLE
此模块无法动态编译？(Unable to compile as dynamic modules?)

### ISSUE THREAD

**Issue by mashirozx** [yaoweibin/ngx_http_substitutions_filter_module#31]
```
# ./configure --prefix=/etc/nginx/ \
            --with-stream \
            --with-http_geoip_module \
            --sbin-path=/usr/sbin/nginx \
            --conf-path=/etc/nginx/nginx.conf \
            --http-log-path=/var/log/nginx/access.log \
            --error-log-path=/var/log/nginx/error.log \
            --add-dynamic-module=/home/sufe/modules/ngx_http_substitutions_filter_module
# make modules
make -f objs/Makefile modules
make[1]: Entering directory '/home/sufe/tengine'
make[1]: Nothing to be done for 'modules'.
make[1]: Leaving directory '/home/sufe/tengine'
```

Tengine version: Tengine/2.3.1
nginx version: nginx/1.16.0

**Comment by mashirozx** [comment]
Fixed in #32

**Comment by cuber** [comment]
duplicated with #19
